scoring/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOSettings.scala

```diff
@@ -28,13 +28,13 @@ case class H2OMOJOSettings(
                             convertUnknownCategoricalLevelsToNa: Boolean = ai.h2o.sparkling.ml.models.H2OMOJOSettings.default.
                               convertUnknownCategoricalLevelsToNa,
                             convertInvalidNumbersToNa: Boolean = ai.h2o.sparkling.ml.models.H2OMOJOSettings.default.convertInvalidNumbersToNa,
-                            namedMojoOutputColumns: Boolean = ai.h2o.sparkling.ml.models.H2OMOJOSettings.default.namedMojoOutputColumns
+                            namedMojoOutputColumns: Boolean = ai.h2o.sparkling.ml.models.H2OMOJOSettings.default.namedMojoOutputColumns,
+                            calculateContributions: Boolean = ai.h2o.sparkling.ml.models.H2OMOJOSettings.default.calculateContributions
```

Comment:
Would it be possible to use an import for `H2OMOJOSettings`?

Reply:
yup, done, thanks!